ddtrace/tracer.py

```diff
@@ -92,6 +98,28 @@
 AnyCallable = TypeVar("AnyCallable", bound=Callable)
 
 
+def _start_appsec_processor():
+    # type: () -> Optional[Any]
+    # FIXME: type should be AppsecSpanProcessor but we have a cyclic import here
```

Comment:
we should be able to get around that by using `"AppsecSpanProcessor"` as the type

Reply:
Tried already, doesn't work: `ddtrace/tracer.py:101: error: Name "AppsecSpanProcessor" is not defined  [name-defined]`